back/src/controllers/artists.js

```diff
@@ -12,11 +13,41 @@ artistsRouter.post('', async (request, response) => {
       return response.status(400).send({ error: 'User already exists' });
 
     const artist = await Artist.create(request.body);
-
     artist.password = undefined;
 
     return response.status(201).send({ artist });
   } catch (error) {
     return response.status(400).json({ message: "Registration failed", error });
   }
-});
\ No newline at end of file
+});
+
+// Get artist information
+artistsRouter.get('/:artistId', async (request, response) => {
+  try{
+    const artist = await Artist.findOne({"_id": request.params.artistId});
+    if (artist) response.send(artist);
+    else throw "Artist doesn't exist in the database";
```

Comment:
Bota um `return response.status(404).json({message: "Could not find the artist"})` aqui ao invés do trhow, pois o código de erro 404 só é usado para not found. Outros tipos de erro são outros códigos.